test-runner-poc/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "runner.js",
   "scripts": {
     "test": "mocha --file lib/runner.js test/crud_tests.js",
+    "test-ci": "mocha --file lib/runner.js ../test/functional/crud_spec_tests.js",
```

Comment:
Let's rename this from `test-ci` to something like `test-crud`. `ci` usually means "continuous integration", which isn't what we're doing here.